### DISCUSSION TITLE
Error installing version 3.13.2-1 on Linux Mint 21.1

### DISCUSSION THREAD

**Discussion by thbeckpr** [GrandOrgue/grandorgue#1734] [General]
Installing grandorgue-wx32_3.13.2-1_amd64.deb threw error: Error: Dependency is not satisfiable: libwxbase3.2-1

**Comment by rousseldenis** [comment]
@thbeckpr Have you tried this one instead :

![image](https://github.com/GrandOrgue/grandorgue/assets/19529533/346c8cb3-398e-4bb9-a710-20477bf1e1e6)

**Comment by thbeckpr** [comment]
That worked, thank you.
